instrumentation/play/play-ws/play-ws-2.1/javaagent/src/latestDepTest/java/io/opentelemetry/javaagent/instrumentation/playws/v2_1/PlayJavaStreamedWsClientTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.playws.v2_1;
+
+import io.opentelemetry.instrumentation.testing.junit.http.HttpClientResult;
+import java.io.IOException;
+import java.net.URI;
+import java.util.Map;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ExecutionException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import play.libs.ws.StandaloneWSClient;
+import play.libs.ws.StandaloneWSRequest;
+import play.libs.ws.StandaloneWSResponse;
+import play.libs.ws.ahc.StandaloneAhcWSClient;
+
+public class PlayJavaStreamedWsClientTest extends PlayWsClientBaseTest<StandaloneWSRequest> {
+
+  private static StandaloneWSClient wsClient;
+  private static StandaloneWSClient wsClientWithReadTimeout;
+
+  @BeforeEach
+  @Override
+  void setup() {
+    super.setup();
+    wsClient = new StandaloneAhcWSClient(asyncHttpClient, materializer);
+    wsClientWithReadTimeout =
+        new StandaloneAhcWSClient(asyncHttpClientWithReadTimeout, materializer);
+    autoCleanup.deferCleanup(wsClient);
+    autoCleanup.deferCleanup(wsClientWithReadTimeout);
+  }
+
+  @AfterEach
+  @Override
+  void tearDown() throws IOException {
+    if (wsClient != null) {
+      wsClient.close();
+    }
+    if (wsClientWithReadTimeout != null) {
+      wsClientWithReadTimeout.close();
+    }
```

Comment:
i wouldn't think these would be needed since you are already registering them the autocleanup